samigo/samigo-app/src/webapp/jsf/delivery/deliverAssessment.jsp

```diff
@@ -33,6 +33,7 @@
     <html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
       <head><%= request.getAttribute("html.head") %>
       <title> <h:outputText value="#{delivery.assessmentTitle}"/> </title>
+      <h:outputText escape="false" value="<script>if(portal===undefined){var portal = {\"siteId\":\"#{delivery.siteId}\"}}</script>" rendered="#{delivery.actionString=='takeAssessmentViaUrl'}" />
```

Comment:
Here you can go:
<script>var portal = portal || { siteId: '#{delivery.siteId}' };</script>